assets/scripts/components/codetabs.js

```diff
@@ -2,7 +2,7 @@ import { getQueryParameterByName } from '../helpers/browser';
 
 const initCodeTabs = () => {
     const codeTabsList = document.querySelectorAll('.code-tabs')
-    const tabQueryParameter = getQueryParameterByName('tab')
+    const tabQueryParameter = getQueryParameterByName('tab') || getQueryParameterByName('tabs')
```

Comment:
in case anyone has pages bookmarked/saved with the `tabs` query parameter, we can use that to set the tab and remove the `tabs` param in favor of `tab`